shared_test.go

```diff
@@ -78,9 +80,15 @@ func (me *logIO) Write(p []byte) (n int, err error) {
 func (me *logIO) Close() (err error) {
 	err = me.proxy.Close()
 	if err != nil {
-		me.t.Logf("%s close : %v\n", me.prefix, err)
+		me.logf("%s close : %v\n", me.prefix, err)
 	} else {
-		me.t.Logf("%s close\n", me.prefix)
+		me.logf("%s close\n", me.prefix)
 	}
 	return
 }
+
+func (me *logIO) logf(format string, args ...interface{}) {
+	me.m.Lock()
+	me.t.Logf(format, args...)
```

Comment:
Still fails - according to https://api.travis-ci.org/jobs/179844976/log.txt?deansi=true. On 1,6, but still.

This one is quite allusive, isn't it? 😆